Traceback (most recent call last):
  File "C:\Users\phoen\atlas-gan\dcgan\utils\preprocess_mnist.py", line 23, in <module>
    with open("configs/dcgan_config.yaml", "r") as file:
FileNotFoundError: [Errno 2] No such file or directory: 'configs/dcgan_config.yaml'
Traceback (most recent call last):
  File "C:\Users\phoen\atlas-gan\dcgan\utils\preprocess_mnist.py", line 23, in <module>
    with open("configs/dcgan_config.yaml", "r") as file:
FileNotFoundError: [Errno 2] No such file or directory: 'configs/dcgan_config.yaml'
